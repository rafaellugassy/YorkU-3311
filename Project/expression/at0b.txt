product

start_of_set_enumeration
int_value(4)
int_value(2)
int_value(3)
int_value(2)
int_value(3)
end_of_set_enumeration

-- duplicates are included for the evaluation
simplify

restart 

sigma

start_of_set_enumeration
int_value(4)
int_value(2)
int_value(3)
int_value(2)
int_value(3)
end_of_set_enumeration

-- duplicates are included for the evaluation
simplify