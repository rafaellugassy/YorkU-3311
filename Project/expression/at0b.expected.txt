  Expression currently specified: ?
  Report: Expression is initialized.
->product
  Expression currently specified: (* ?)
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: (* {?})
  Report: OK.
->int_value(4)
  Expression currently specified: (* {4, ?})
  Report: OK.
->int_value(2)
  Expression currently specified: (* {4, 2, ?})
  Report: OK.
->int_value(3)
  Expression currently specified: (* {4, 2, 3, ?})
  Report: OK.
->int_value(2)
  Expression currently specified: (* {4, 2, 3, 2, ?})
  Report: OK.
->int_value(3)
  Expression currently specified: (* {4, 2, 3, 2, 3, ?})
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: (* {4, 2, 3, 2, 3})
  Report: OK.
->simplify
  Expression currently specified: (* {4, 2, 3, 2, 3})
  Report: 144
->restart
  Expression currently specified: ?
  Report: OK.
->sigma
  Expression currently specified: (+ ?)
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: (+ {?})
  Report: OK.
->int_value(4)
  Expression currently specified: (+ {4, ?})
  Report: OK.
->int_value(2)
  Expression currently specified: (+ {4, 2, ?})
  Report: OK.
->int_value(3)
  Expression currently specified: (+ {4, 2, 3, ?})
  Report: OK.
->int_value(2)
  Expression currently specified: (+ {4, 2, 3, 2, ?})
  Report: OK.
->int_value(3)
  Expression currently specified: (+ {4, 2, 3, 2, 3, ?})
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: (+ {4, 2, 3, 2, 3})
  Report: OK.
->simplify
  Expression currently specified: (+ {4, 2, 3, 2, 3})
  Report: 14
